generator/tools/templates/react/vite.config.ts

```diff
@@ -49,13 +49,21 @@ const psibase = (appletContract: string) => {
                         },
                     },
                     resolve: {
-                        alias: {
-                            "/common/iframeResizer.contentWindow.js":
-                                path.resolve(
+                        alias: [
+                            {
+                                find: "/common/iframeResizer.contentWindow.js",
+                                replacement: path.resolve(
                                     "../../CommonSys/common/thirdParty/src/iframeResizer.contentWindow.js"
                                 ),
-                            "/common": path.resolve("../../CommonSys/common"),
-                        },
+                            },
+                            {
+                                // bundle non-external (above) common files except fonts (which should only be referenced)
+                                find: /^\/common(?!\/(?:fonts))(.*)$/,
+                                replacement: path.resolve(
+                                    "../../CommonSys/common$1"
+                                ),
+                            },
```

Comment:
At first, I tried simply adding the font files to the `rollupOptions.external` array above. This works fine for js modules. But I learned that `external` is _only_ for js modules and applies that mime type to any files in that array. It works in dev (hmr) mode, but not when running from a build; some browsers reject the css and fonts because they're improperly mime typed as javascript modules.